synapse/tools/pkg/build_docs.py

```diff
@@ -0,0 +1,186 @@
+import os
+import sys
+import shutil
+import asyncio
+import logging
+import argparse
+import subprocess
+
+import regex as re
+
+import synapse.exc as s_exc
+import synapse.common as s_common
+
+import synapse.lib.coro as s_coro
+import synapse.lib.output as s_output
+
+import synapse.tools.rstorm as s_rstorm
+import synapse.tools.autodoc as s_autodoc
+
+logger = logging.getLogger(__name__)
+
+logging.getLogger('vcr').setLevel(logging.WARNING)
```

Comment:
This is unexpected.

Reply:
That's an artifact from porting this script over.